sbt-plugin-test/multiTest/js/src/test/scala/sbttest/multitest/JUnitUtil.scala

```diff
@@ -3,17 +3,16 @@ package sbttest.multitest
 import org.scalajs.junit.JUnitTestBootstrapper
 import org.junit.Assert.fail
 
-import scalajs.js
+import scala.scalajs.reflect.Reflect
 
 object JUnitUtil {
   private final val bootstrapperSuffix = "$scalajs$junit$bootstrapper"
 
   def loadBootstrapper(classFullName: String): JUnitTestBootstrapper = {
     val fullName = s"$classFullName$bootstrapperSuffix"
     try {
-      fullName.split('.').foldLeft(js.Dynamic.global) { (obj, n) =>
-        obj.selectDynamic(n)
-      }.apply().asInstanceOf[JUnitTestBootstrapper]
+      val modClass = Reflect.lookupLoadableModuleClass(fullName + "$").get
+      modClass.loadModule().asInstanceOf[JUnitTestBootstrapper]
```

Comment:
This one is intentional. I want to test that JUnit indeed works through `Reflect`, and not through the fall-back of `exportsNamespace`. This is also the reason this specific change was applied in the same commit that changes JUnit to use `@EnableReflectiveInstantiation`, whereas all changes to `TestUtils` are done in the commit "Prepare the testing interface ...".